components/layout_2020/flow/inline.rs

```diff
@@ -2405,6 +2405,7 @@ impl<'a> ContentSizesComputation<'a> {
             had_non_whitespace_content_yet: false,
             linebreaker: None,
             ending_inline_pbm_stack: Vec::new(),
-        }.traverse(inline_formatting_context)
+        }
+        .traverse(inline_formatting_context)
```

Comment:
I think this change is accidental.

Reply:
This is `./mach fmt` doing. I noticed this in another change as well. Would you like me to revert it? But mach fmt will suggest it again.

Reply:
Huh! Sorry. Nope it's fine to leave it as-is.